### PR TITLE
Feat/tt 511 new gdc release node

### PR DESCRIPTION
**Pull request by wwysoc2**
https://jira.opensciencedatacloud.org/browse/TT-511

Added new gdc_release node and modified _terms and valid json example to fit.

### CODE REVIEW

File: gdcdictionary/schemas/gdc_release.yaml [hunk 1]
```diff
@@ -0,0 +1,80 @@
+$schema: "http://json-schema.org/draft-04/schema#"
+
+id: "gdc_release"
+title: GDC Release 
+type: object
+namespace: http://gdc.nci.nih.gov
+category: administrative 
+program: '*'
+project: '*'
```
Comment (christiandompierre) [inline gdcdictionary/schemas/gdc_release.yaml]: Can we restrict these?

File: gdcdictionary/schemas/gdc_release.yaml [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+description: >
+  Internal node to store different GDC releases.
+additionalProperties: false
+submittable: false
+validators: null
+
+systemProperties:
+  - id
+  - project_id
+  - state
```
Comment (christiandompierre) [inline gdcdictionary/schemas/gdc_release.yaml]: We don't need state afaik

File: gdcdictionary/schemas/gdc_release.yaml [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+    $ref: "_definitions.yaml#/UUID"
+    systemAlias: node_id
+  submitter_id:
+    type:
+      - string
+    description: >
+      A project-specific identifier for a node. This property is the calling card/nickname/alias for
+      a unit of submission. It can be used in place of the UUID for identifying or recalling a node.
+  state:
+    $ref: "_definitions.yaml#/state"
```
Comment (christiandompierre) [inline gdcdictionary/schemas/gdc_release.yaml]: we don't need state afaik

File: gdcdictionary/schemas/gdc_release.yaml (hunk 1 above)
Comment (wwysoc2) [reply gdcdictionary/schemas/gdc_release.yaml]: Every schema has those same values for program and project. I'm guessing program would be "GDC", but would project be "MISC" or "GDC-MISC".  Any ideas @madopal ?

File: gdcdictionary/schemas/gdc_release.yaml (hunk 2 above)
Comment (wwysoc2) [reply gdcdictionary/schemas/gdc_release.yaml]: Removed

File: gdcdictionary/schemas/gdc_release.yaml (hunk 3 above)
Comment (wwysoc2) [reply gdcdictionary/schemas/gdc_release.yaml]: Removed

File: gdcdictionary/schemas/gdc_release.yaml (hunk 1 above)
Comment (christiandompierre) [reply gdcdictionary/schemas/gdc_release.yaml]: My guess is that project would be "MISC". This is fine for now, we can test it on our cluster.

Comment (DrSpez) [reply gdcdictionary/schemas/gdc_release.yaml]: Yeah, it says `GDC-MISC` in the ticket https://jira.opensciencedatacloud.org/browse/TT-511
Don't forget to restrict it before the dictionary release though

Comment (DrSpez) [reply gdcdictionary/schemas/gdc_release.yaml]: Meaning, `program=GDC`, `project=MISC`

Comment (wwysoc2) [reply gdcdictionary/schemas/gdc_release.yaml]: Project has been restricted to GDC-MISC.

File: gdcdictionary/schemas/gdc_release.yaml [hunk 4]
```diff
@@ -0,0 +1,77 @@
+$schema: "http://json-schema.org/draft-04/schema#"
+
+id: "gdc_release"
+title: GDC Release 
+type: object
+namespace: http://gdc.nci.nih.gov
```
Comment (dmiller15) [inline gdcdictionary/schemas/gdc_release.yaml]: Change to `https://gdc.cancer.gov/`

File: gdcdictionary/schemas/gdc_release.yaml [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+  - project_id
+  - created_datetime
+  - updated_datetime
+
+links:
+  - name: cases
+    backref: gdc_releases
+    label: describes
+    target_type: case
+    multiplicity: many_to_one
```
Comment (dmiller15) [inline gdcdictionary/schemas/gdc_release.yaml]: Wouldn't many cases be in a single release? Also how can a case be in multiple releases?

Comment (DrSpez) [reply gdcdictionary/schemas/gdc_release.yaml]: Scratch that. There was decision made on Friday to make a new `Root` node to which this node will be directly connected as well as all `Program` nodes. Check with @christiandompierre @philloooo for more info

Comment (DrSpez) [reply gdcdictionary/schemas/gdc_release.yaml]: * the part about connecting `Program` is being debated

**Comment by madopal** [comment]
I'm assuming we're dealing with this in datamodel as per discussions earlier this week. I'll leave the branch just in case, but I'm closing this PR.

**Pull request closed without merging**